Sources/TuistAutomation/Utilities/Formatter.swift

```diff
@@ -0,0 +1,20 @@
+import Foundation
+import TuistCore
+import TuistSupport
+
+public protocol Formatting {
```

Comment:
I don't think this has to be public ... or am I mistaken?

Reply:
You are correct. Great catch, thank you! 👍